ComfortZoneApp/app/src/main/java/com/example/comfortzone/fragments/FlightFragment.java

```diff
@@ -226,18 +227,21 @@ public void onButtonChecked(MaterialButtonToggleGroup group, int checkedId, bool
         });
     }
 
+    private void updateViewsList(List<WeatherData> cityList) {
+        CityListViewFragment cityListViewFragment = (CityListViewFragment) fragmentManager.findFragmentByTag(TAG_LIST_VIEW);
+        cityListViewFragment.onCityListUpdated(cityList);
```

Comment:
does it have to be just for ListView? I think you can be a bit more generic and cast it to `UpdateCityListCallback`, so you can do it for both list and map